lib/bots/syncbot.ts

```diff
@@ -102,9 +115,16 @@ export class SyncBot extends ProcBot {
 							};
 						});
 					}
-					logger.log(LogLevel.DEBUG, `---> Creating thread '${firstLine}' on ${toText}.`);
-					// Create a thread if the quest for connections didn't find any
-					return SyncBot.createThreadAndConnect(to, messenger, data);
+					if (!threadId && _.includes(actions, MessengerAction.CreateThread)) {
+						logger.log(LogLevel.INFO, `---> Creating thread '${firstLine}' on ${toText}.`);
+						// Create a thread if the quest for connections didn't find any
+						return SyncBot.createThreadAndConnect(to, messenger, data);
+					}
+					// Pass on that this has no connected thread
+					return Promise.resolve({
```

Comment:
Don't need a `Promise.resolve()` here as you're inside a BlueBird `.then()` clause so the return will automatically default to resolve.